utilities/vcf2json.py

```diff
@@ -70,7 +70,7 @@ def _valid_record(record, genomic_source_class, sample_position):
            genomic_source_class.lower() == Genomic_Source_Class.GERMLINE.value.lower()):
             return False
     else:
-        if(not all(alt is None or alt.type in ['SNV', 'MNV'] or '*' not in str(alt)
+        if(not all(alt is None or ((alt.type in ['SNV', 'MNV'] or '*' not in str(alt)) and str(alt).isalpha())
```

Comment:
This won't work for angle-bracketed tokens. Refer to the exclusion criteria for ALT [here](https://github.com/elimuinformatics/vcf2fhir/blob/master/docs/Manual.md#exclude-vcf-rows)
<img width="627" alt="Screenshot 2022-10-05 at 3 46 29 PM" src="https://user-images.githubusercontent.com/47586886/194037615-02feb6c6-8a52-4f7d-9ec2-801e094e2e37.png">

We can have a common regular expression for simple and structural variants.